release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -503,6 +555,24 @@ The `--filter-by-os` flag is also now deprecated.
 
 When using installer-provisioned installation on bare metal nodes, {product-title} 4.6 required providing two IP addresses from the `baremetal` network to the `provisioningHostIP` and `bootstrapProvisioningIP` configuration settings when deploying without a `provisioning` network. These IP addresses and configuration settings are no longer required in {product-title} 4.7 when using installer provisioned infrastructure on bare metal nodes and deploying without a `provisioning` network.
 
+==== Removed images
+
+The following images were removed in this release:
```

Comment:
Perhaps a *little* more elaboration here @rolfedh, like 

The following images are no longer included in the samples imagestreams provided with OCP

or some such

the list itself matches what I have from our recent email exchange ... so cool deal there

thanks @rolfedh !!

Reply:
@gabemontero  I tried to make this change but am having merge issues. Doubly complicated because every other OpenShift writer is submitting changes to this file. I'll add that phrase in a new PR after this PR merges.  RHDEVDOCS-2653 - Tweak image removal phrase in 4.7 RNs
Update to:
```
==== Removed images from samples imagestreams
The following images are no longer included in the samples imagestreams provided with {product-title}:
```

Reply:
https://issues.redhat.com/browse/RHDEVDOCS-2653

Reply:
10-4 thanks @rolfedh